docs/server-setup.md

```diff
@@ -55,9 +55,9 @@ AzerothCore needs certain files extracted from the World of Warcraft client. You
 
 By default you will compile your core with tools and you will get the following executable files: **mapextractor**, **vmap4extractor**, **vmap4assembler**, **mmaps_generator** (.exe on windows).
 
-Place the files with your World of Warcraft binary (wow.exe on windows) and run them.
+Place the files in the same directory as your World of Warcraft binary (wow.exe on windows) and run them one after the other.
 
-After extracting all necessary files, create a folder called **Data** within the **RelWithDebInfo** or **Debug** directory and place the files in there. Alternatively you can specify a different directory where you want to keep them by changing DataDir value in the worldserver.conf file.
+After extracting all necessary files, create a folder called **Data** (as created in step 2 of the previous section) and place files in there. Alternatively you can specify a different directory where you want to keep them by changing DataDir value in the [DataDir](#updating-datadir) worldserver.conf file.
```

Comment:
You can't point to a previous step that isn't within the scope of the topic.
`##Download` & `##Extractors (Not needed if you downloaded the files above)` are two different topics.

Also keep the RelWithDebInfo and Debug stuff.